Dockerfile

```diff
@@ -0,0 +1,96 @@
+
+# Adds DeepX Gazebo Rendering image.
+#
+# Arguments:
+# - BASE_IMAGE: Base image.
+# - TAG: Tag of the BASE_IMAGE.
+#
+# Docker commands:
+# - To build the image:
+#   docker build --build-arg BASE_IMAGE=ghcr.io/deepx-inc/base_images --build-arg TAG=foxy -t deepx-gz-rendering .
+# - To publish the image:
+#   docker push ghcr.io/deepx-inc/deepx-gz-rendering
+
+
+ARG BASE_IMAGE
+ARG TAG
+ARG FROM_SOURCE
+FROM ${BASE_IMAGE}:${TAG} AS DEEPX_GZ_RENDERING
+
+# ---BUILD TOOLS---
+RUN apt -qq update \
+    && apt -qq install -y --no-install-recommends \
+    cmake \
+    ffmpeg \
+    freeglut3-dev \
+    g++-8 \
+    git \
+    gnupg \
+    libfreeimage-dev \
+    libglew-dev \
+    libogre-2.1-dev \
+    libxi-dev \
+    libxmu-dev \
+    ninja-build \
+    pkg-config \
+    redis-server \
+    redis-tools \
+    software-properties-common \
+    lsb-release \
+    wget \
+    curl
+
+# ---Temporary solution to stop flikering in gazebo
+# Before mesa version 22.0.2, there was no flickering issue.
+# After doing apt update && apt upgrade, the flickering appears.
+# This flickering is observed in mesa version 22.1.1
+# [TODO] Remove this apt-mark hold, after the issue is solved.
+RUN apt-mark hold \
```

Comment:
We don't need to hold these packages.

Reply:
If I understand correctly: deleted the whole `RUN ...`.